src/metabase/query_processor/pivot.clj

```diff
@@ -32,7 +32,7 @@
   "Add the grouping field and expression to the query"
   [query breakout bitmask]
   (-> query
-      (assoc-in [:query :breakout] breakout)
+      (assoc-in [:query :breakout] (conj (seq breakout) [:expression "pivot-grouping"]))
```

Comment:
`(conj (seq breakout)...` feels wrong. I think one of the values for `breakout` is a vector since it was being appended for one of the breakouts in the test. Is there a better way to append/prepend consistently?

Reply:
`cons` always prepends, but returns a seq which is not ideal (in general the schema calls for a vector and as you've just experienced we don't always gracefully handle other seq types). If you want to consistently append, we usually use either `(concat seq [value])`, or `(conj (vec seq) breakout)`. In general I personally prefer `concat` as it's the most explicit, but in this case explicitly casting to vector is being nice to everyone downstream. 

Having said all that, if this is only to make the Testing Gods happy, I'd be tempted to just change the tests and sneak in something along the lines of `(update :breakout sort)`, rather than complicating things here.